site/en/guides/border-router/mdns-discovery.md

```diff
@@ -0,0 +1,123 @@
+
+There are several ways to discover your Thread network.
+
+## DNS Service Discovery (DNS-SD)
+
+Use `dns-sd` to browse for a service on `raspberrypi.local.:49155`:
+
+```
+$ dns-sd -L "OpenThread_BorderRouter" _meshcop._udp local
+```
+
+<pre>
+Lookup OpenThread_BorderRouter._meshcop._udp.local
+DATE: ---Tue 16 Nov 2021---
+13:31:03.197  ...STARTING...
+13:31:03.350  OpenThread_BorderRouter._meshcop._udp.local. can be reached at raspberrypi.local.:49155 (interface 3)
+</pre>
+
+Get the IP addresses associated with your OTBR domain name:
+
+```
+$ dns-sd -G v4/v6 {raspberrypi.local}
+```
+
+<pre>
+DATE: ---Tue 16 Nov 2021---
+14:21:29.485  ...STARTING...
+Timestamp     A/R Flags if Hostname           Address                                      TTL
+14:21:29.486  Add     3  3 raspberrypi.local. FDDE:AD11:11DE:0000:74D0:6FC9:6BE6:3582%&lt;0&gt;  120
+14:21:29.486  Add     3  3 raspberrypi.local. FD00:0000:0000:0000:74D0:6FC9:6BE6:3582%&lt;0&gt;  120
+14:21:29.486  Add     3  3 raspberrypi.local. FE80:0000:0000:0000:74D0:6FC9:6BE6:3582%eth0 120
+14:21:29.486  Add     3  3 raspberrypi.local. FE80:0000:0000:0000:287F:87CA:F4B3:498A%eth0 120
+14:21:29.486  Add     2  3 raspberrypi.local. 192.168.0.10                                 120
+</pre>
+
+Ping your IP address. From the `dns-sd` results, choose an address that's
+reachable from your network, for example the global `<0>` address
+**FD00:0000:0000:0000:74D0:6FC9:6BE6:3582**:
+
+```
+$ ping -6 FD00:0000:0000:0000:74D0:6FC9:6BE6:3582
+
+```
+
+<pre>
+PING FD00:0000:0000:0000:74D0:6FC9:6BE6:3582(fd00::74d0:6fc9:6be6:3582) 56 data bytes
+64 bytes from fd00::74d0:6fc9:6be6:3582: icmp_seq=1 ttl=64 time=27.1 ms
+64 bytes from fd00::74d0:6fc9:6be6:3582: icmp_seq=2 ttl=64 time=3.18 ms
+64 bytes from fd00::74d0:6fc9:6be6:3582: icmp_seq=3 ttl=64 time=2.76 ms
+</pre>
+
+## Avahi utilities
+
+Install `avahi-daemon` and `avahi-utils`:
+
+```
+$ sudo apt-get install -y avahi-daemon avahi-utils
+```
+
+Start the `avahi-daemon`:
+
+```
+$ sudo service avahi-daemon start
+```
+
+Use `avahi-browse`. To resolve IPv6 only, edit `/etc/avahi/avahi-daemon.conf`
+and set `use-ipv4=no` and `use-ipv6=yes`.
+
+```
+$ avahi-browse -r -t _meshcop._udp
+```
+
+<pre>
++ eth0 IPv6 OpenThread_BorderRouter   _meshcop._udp        local
+= eth0 IPv6 OpenThread_BorderRouter   _meshcop._udp        local
+   hostname = [raspberrypi.local]
+   address = [192.168.0.10]
+   port = [49155]
+   txt = []
+</pre>
+
+## mDNS apps
+
+Search [Google Play](https://play.google.com/store/search?q=mDNS%20discovery&c=apps) for mDNS
+discovery apps, for example:
+
+* [Service Browser](https://play.google.com/store/apps/details?id=com.druk.servicebrowser) for Android
+* [Discovery - DNS-SD Browser](https://apps.apple.com/app/discovery-dns-sd-browser/id305441017) for iOS
+
+> Note: Make sure that your mobile device is on the same network as your Thread
+network, and connected to Wi-Fi 2.4GHz.
```

Comment:
Doesn't it also work for Wi-Fi 5G?

Reply:
I had my phone on 5G and had issues w/the Service Browser app and OT Commissioner for Android. Once I switched, everything worked perfectly. Could that be an issue with my network? I made some updates to say that if you're having issues and you're on 5G, try switching.

Reply:
Are your BR also in 5G WiFi network when your phone is in 5G WiFi?

Reply:
Ah - I had my BR on Ethernet; same network as phone. I haven't tried my BR w/WiFi yet; I would like to. The strange thing was that nothing showed up in the Service Browser app either - none of my services - I have a lot :) Once I made the switch, all services appeared. I'll have to try some things out again...maybe do some tests. It's also a new phone; I haven't used it too much yet.